x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -35,10 +36,10 @@ const querystringStringify: <ExpectedType extends object, ArgType>(
 type EndpointDetailsUrlProps = Omit<EndpointIndexUIQueryParams, 'selected_endpoint'> &
   Required<Pick<EndpointIndexUIQueryParams, 'selected_endpoint'>>;
 
-export const getEndpointListPath = (
+export function getEndpointListPath(
```

Comment:
What happen on these `const`? Editor auto-reformat?

(FYI: it was a while back, but I seem to remember Kibana dev. practices advocating for use of arrow functions whenever possible)

Reply:
> (FYI: it was a while back, but I seem to remember Kibana dev. practices advocating for use of arrow functions whenever possible)

This is true which is why we switched over to arrows over functions

Reply:
Nope, I don't really use auto reformatting. To be honest it was intentional and triggered the right discussion ;)

I can of course revert back. My reasoning was that function declaration has better readability and better properties than const statements. I think function declaration conveys the intent way more clear - you are declaring a function, while const is to declare constant. Function declaration is also subject to function hoisting which is also a really nice property.

I checked Kibana contribution guide and the only thing in there I could find was:

`Prefer arrow function over function expressions`

I think that's where the confusion comes from - the above mentions function expression, while the code you commented on is function declaration. Function expressions are more of a form:

`arr.map(function(item) {...});`

or

`const func = function() {...};`

And I totally agree that there it makes total sense to use lambda's. But what is the benefit of using lambda's instead of function declarations? The Kibana contribution guides is full of examples of function declarations.

In any case if you think it's crucial to revert back to const let me know, I can easily do that.

Reply:
@efreeti I think we should keep with the lambda functions since it is an established pattern in the code base currently.  We can continue the conversation outside of the PR, but I don't want it to hold this up.

Reply:
@efreeti you are correct in deed that the guideline refers to expressions - I myself perhaps overlooked that 😉 .

Discussions like this is why I ❤️ code formatters - we all have our own personal (opinionated) preferences for code readability (_maybe one day I'll share with you how I used to align my variable declarations_ 😄 ). Code formatters just take care of that by "forcing" a structure across the board, which one will eventually adopt. I too used to be a heavy user of `function` declarations for the same purpose as yours, but have shifted my mindset towards arrow functions - mainly because it alleviates the historical issues with `this`. Again - personal preference. There is nothing functionally concerning about this code block and that is more important aspect of code review (IMO). Just realize that we will have different patterns (since the formatter does not address this) and you will likely encounter lots of arrow function usage through other areas of kibana.

Reply:
@paul-tavares @efreeti apologies, I realize I was unclear above - I don't think the conversation should hold up this PR, but I still think we should keep with the lamdas until we have that conversation since it's an established pattern.  I'll approve after that.  Thanks for the work on this Bohdan

Reply:
reverted